LordUche/ireporter-react
# #164034242 Add Login and Signup Functionalities

#### What does this PR do?
Add authentication functionalities
#### Description of Task to be completed?
- create login and signup form components
- create auth reducer and action creators
- connect components to redux store
- write tests
#### How should this be manually tested?
- ensure that build is passing on `Travis CI`
- view the [review app](https://uche-ireporter-react-stag-pr-3.herokuapp.com) and verify that everything looks good
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
[#164034242](https://www.pivotaltracker.com/story/show/164034242)
#### Screenshots (if appropriate)
N/A
#### Questions:
N/A
